static_src/stores/service_instance_store.js

```diff
@@ -17,6 +17,7 @@ export const OPERATION_DELETING = 'deleting';
 const OPERATION_PROCESSING = 'processing';
 export const OPERATION_RUNNING = 'running';
 const OPERATION_INACTIVE = 'inactive';
+export const CREATED_NOTIFICATION_TIME_MS = 2000;
```

Comment:
FWIW, I think 2 seconds is a bit short. Not sure if we have better guidance around this, but I think 3-5 seconds would be better.